api/app/services/demonstration/data_loader.rb

```diff
@@ -70,6 +77,25 @@ def create_pages
       end
     end
 
+    def ensure_settings
+      settings = Settings.instance
+      ENV_SETTINGS.each do |key, values|
+        values.each do |value|
+          env_setting = "MANIFOLD_SETTING_#{key}_#{value}".upcase
+          settings[key][value] = settings[key][value] ||= ENV[env_setting]
```

Comment:
You didn't quite get my meaning here. This line should be:

`settings[key][value] ||= ENV[env_setting]`